UI/UserProfile.html

```diff
@@ -0,0 +1,208 @@
+											<p>
+													Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+													tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+													quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+													consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+												</p>
+										<b>1st, Jan 2019</b>
+									</span>
+								</div>
+							</div>
+						</div>
+						<div class="feed-card-footer">
+							<small>PEOPLE DEMOCRATIC PARTY</small>
+						</div>
+					</div>
+				</div>
+				<div class="feed-area">
+					<div class="feed-card">
+						<div class="feed-body">
+							<div class="left">
+								<div class="vote">
+									</BR>
+									<img src="assets/images/check.png" alt="check" height="30px" width="30px">
+								</div>
+							</div>
+							<div class="right">
+								<h3 class="feed-card-title">
+									<a href="./questions.html">
+										TAIWO OYINDAMOLA EZEKIEL
+									</a>
+								</h3>
+								<div class="feed-body-content">
+									<span class="feed-date">
+											<p>
+													Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+													tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+													quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+													consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+												</p>
+										<b>1st, Jan 2019</b>
+									</span>
+								</div>
+							</div>
+						</div>
+						<div class="feed-card-footer">
+							<small>PEOPLE DEMOCRATIC PARTY</small>
+						</div>
+					</div>
+				</div>
+				<div class="feed-area">
+					<div class="feed-card">
+						<div class="feed-body">
+							<div class="left">
+								<div class="vote">
+									</BR>
+									<img src="assets/images/check.png" alt="check" height="30px" width="30px">
+								</div>
+							</div>
+							<div class="right">
+								<h3 class="feed-card-title">
+									<a href="./questions.html">
+										TAIWO OYINDAMOLA EZEKIEL
+									</a>
+								</h3>
+								<div class="feed-body-content">
+									<span class="feed-date">
+											<p>
+													Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+													tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+													quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+													consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+												</p>
+										<b>1st, Jan 2019</b>
+									</span>
+								</div>
+							</div>
+						</div>
+						<div class="feed-card-footer">
+							<small>PEOPLE DEMOCRATIC PARTY</small>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div> <BR></BR>
+		<footer class="politico-footer">
+			<p>Politico.  &copy; 2019</p>
+		</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks